CHANGELOG.md

```diff
@@ -48,12 +48,17 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Added CodeQL analysis to esp32, stm32, pico, and wasm workflows
 - Added Function.ex and Protocol.ex improving Elixir 1.18 support
 - Added WiFi support for ESP32P4 via esp-wifi-external for build with ESP-IDF v5.4 and later
+- Added `system_info/1` key `atomvm_free_heap_size` support for STM32
+- Added `system_info/1` key `atomvm_minimum_free_size` support for STM32
 
 ### Changed
 
 - Removed `externalterm_to_term_copy` added in [0.6.5] and introduced flags to `externalterm_to_term` to perform copy.
 - Release images for ESP32 chips are built with ESP-IDF v5.4
 - ESP32: SPI peripheral defaults to `"spi2"` instead of deprecated `hspi`
+- Add transitional ESP32 `system_info/1` memory keys; `atomvm_free_heap_size` for `esp32_free_heap_size` and
+`atomvm_minimum_free_size` for `esp32_minimum_free_size` to match newly added STM32 memory keys, with
+warning to update applications when the old keys are used.
```

Comment:
Let's rephrase this in a maybe clearer way: let's write that esp32_free_heap_size and esp32_minimum_free_size have been renamed, but for a while the old names will remain available (but deprecated)